database/db.go

```diff
@@ -50,3 +51,23 @@ func createDSN() string {
 
 	return "host=" + host + " port=" + port + " user=" + username + " password=" + password + " dbname=" + dbname
 }
+
+// createConfig creates a Gorm config depending on the environment variables.
+func createConfig() *gorm.Config {
+	env, found := os.LookupEnv("ENVIRONMENT")
+	if !found {
+		log.Println("ENVIRONMENT environment variable not found")
+		return &gorm.Config{}
+	}
+
+	switch env {
+	case "TEST":
+		return &gorm.Config{Logger: logger.Default.LogMode(logger.Silent)}
+	case "DEV":
+		return &gorm.Config{Logger: logger.Default.LogMode(logger.Info)}
+	case "PROD":
+		return &gorm.Config{Logger: logger.Default.LogMode(logger.Error)}
+	default:
+		return &gorm.Config{}
```

Comment:
Maybe log that `env` has an invalid value

Reply:
Add env cases to an immuable `struct` or `slice`, so you can list them in the `default:`, preventing others from having to look into the code